autobahn/wamp/message.py

```diff
@@ -2023,7 +2065,8 @@ def parse(wmsg):
                     topic=topic,
                     enc_algo=enc_algo,
                     enc_key=enc_key,
-                    enc_serializer=enc_serializer)
+                    enc_serializer=enc_serializer,
+                    retained=retained)
```

Comment:
Okay, non-keyword based calling was my only concern. If we're not worried about that, I can relocate them to where it makes sense.